src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -355,6 +356,7 @@ public void initConfig(Config config) {
     config.addKey("documentum.maxHtmlSize", "1000");
     config.addKey("adaptor.caseSensitivityType",
         "everything-case-sensitive");
+    config.addKey("documentum.updatedDocumentsQuery", "");
```

Comment:
s/updated/modified/, for consistent public nomenclature (e.g., getModifiedDocIds).